ai_feedback/models/CodeLlamaModel.py

```diff
@@ -86,14 +95,13 @@ def _get_question_contents(
 
         for file_path in assignment_files:
             if (
-                not file_path.endswith(".txt")
-                or "error_output" in file_path
-                or file_path.endswith(".DS_Store")
+                file_path.suffix == '.txt' or
```

Comment:
It is okay to change the attributes used on `file_path` because the type has changed. However, `not` should still be there (otherwise the logic is changed); and `or` should not be moved.

Please double-check the conditions you modified throughout to ensure that the only changes are ones that flow directly from the type change of `file_path`.